### DISCUSSION TITLE
Recognition of slightly italic digital numbers sometimes fails.

### DISCUSSION THREAD

**Discussion by Brakelmann** [jomjol/AI-on-the-edge-device#1328] [General]
**Describe the bug**
Recognition of slightly italic digital numbers sometimes fails.
The recognition of the third number after the comma often fails.

**Version**
V 11.3.1

**Screenshots**
![image](https://user-images.githubusercontent.com/32041481/201299158-4fa026aa-4f5a-4586-b028-aa21d9cb20ed.png)

Recognition:
![image](https://user-images.githubusercontent.com/32041481/201299235-9ef97d80-1bf0-4d86-a66b-87f9ffdbea4a.png)

Is there any chance to get a better recognition rate?

Thanks in advance for any support.

**Comment by caco3** [comment]
There are already similar digits in the training data,  see https://jomjol.github.io/neural-network-digital-counter-readout/
But they are less "italic".

You could try to rotate it a few degrees anti-clock-wise to compensate the tilting and use different y-values for the ROIs

**Reply by Brakelmann** [reply]
Thanks for you hint.
I gave it a try with 2,3 and 4 degrees anti-clock- wise.
Unfortunately without success.
It is recognizing the 0 alway as 7.

Any other idea?

Thanks, Brakelmann.

**Reply by caco3** [reply]
can you show another screenshot?

You also can record your data and send them so we can add them to the training data

**Reply by Brakelmann** [reply]
Sure attached the screenshot with 3 degrees rotated anti clock wise
![Bildschirmfoto 2022-11-11 um 17 23 05](https://user-images.githubusercontent.com/32041481/201385195-63da7ec6-2f0d-43c7-bae9-d77d92a81ec3.png)

And digits with its recognition:

![Bildschirmfoto 2022-11-11 um 17 23 19](https://user-images.githubusercontent.com/32041481/201385273-f8a5ecf2-94d3-4579-a232-7a2935e3c98e.png)

I will try to record the digits and provide the grafics.
Would the last digit be sufficient?

**Reply by caco3** [reply]
try to tilt it more, so the numbers are vertically.

And maybe you have to make the ROIs smaller as well, see here: https://github.com/jomjol/AI-on-the-edge-device/wiki/ROI-Configuration#4-model-selection

**Reply by Brakelmann** [reply]
Yipiie I got a first success.
Tilted to 6 degrees and made the ROIs a bit smaller.

Attached the screenshots with success.
I will keep you updated if it is a stable recognition.
![Bildschirmfoto 2022-11-11 um 17 40 05](https://user-images.githubusercontent.com/32041481/201387889-062aa592-8c80-44f5-8e8e-b1568ffb2dd4.png)

![Bildschirmfoto 2022-11-11 um 17 40 20](https://user-images.githubusercontent.com/32041481/201387927-a43dc82c-2cd3-484a-a43a-a94617d3be0a.png)


Thanks, you saved my weekend.
